packages/web-vite/src/components/LeftNavigation.tsx

```
@@ -0,0 +1,146 @@
+      )}
+
+      {/* Left navigation panel */}
+      <nav className={`left-navigation ${isMobileMenuOpen ? 'open' : ''}`}>
+        {/* Close button for mobile */}
+        <button
+          className="nav-close-btn"
+          onClick={() => setIsMobileMenuOpen(false)}
+          aria-label="Close navigation menu"
+        >
+          ✕
+        </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)